components/tools/OmeroWeb/omeroweb/api/api_query.py

```diff
@@ -20,12 +20,36 @@
 """Helper functions for views that handle object trees."""
 
 
-from omero.rtypes import unwrap
+from omero.rtypes import unwrap, wrap
+from omero.sys import ParametersI
 
 from api_marshal import marshal_objects
 from copy import deepcopy
 
 
+def get_child_counts(conn, link_class, parent_ids):
+    """
+    Count child links for the specified parent_ids.
+
+    @param conn:        BlitzGateway
+    @param link_class:  E.g. 'ProjectDatasetLink'
```

Comment:
This is really the start of a sentence, similar to lines above and below that are all capitalised, and I think it's easier to read like that.